sites/bizbash.com/server/routes/website-section.js

```diff
@@ -40,17 +34,17 @@ module.exports = (app) => {
   }));
 
   app.get('/:alias(production-strategy)', withWebsiteSection({
-    template: productionStrategyTemplate,
+    template: sectionWithListsTemplate,
```

Comment:
@Shinsina 

if we are consolidating all this we should be able to do something like this as well.

```

const sectionWithListAliases = [
  'production-strategy',
  'catering-design',
  'event-tech-virtual',
  'bizbash-lists',
  'hubilo',
  'on-demand',
  'meetings-trade-shows',
];

sectionWithListAliases.forEach((alias) => {
    app.get(`/:alias(${alias})`, withWebsiteSection({
      template: sectionWithListsTemplate,
      queryFragment,
    }));
  });
  ```